google/cloud/spanner/samples/samples.cc

```diff
@@ -88,6 +88,31 @@ void ListInstancesCommand(std::vector<std::string> const& argv) {
   ListInstances(std::move(client), argv[0]);
 }
 
+//! [instance-test-iam-permissions]
+void InstanceTestIamPermissions(
+    google::cloud::spanner::InstanceAdminClient client,
+    std::string const& project_id, std::string const& instance_id) {
+  google::cloud::spanner::Instance in(project_id, instance_id);
+  auto actual = client.TestIamPermissions(in, {"spanner.databases.list"});
+  if (!actual) throw std::runtime_error(actual.status().message());
+
+  char const* msg = actual->permissions().empty() ? "does not" : "does";
+
+  std::cout << "The caller " << msg
+            << " have permission to databases on the Cloud Spanner instance "
```

Comment:
"permission to databases" sounds like it's missing a verb? Maybe "list"?

Reply:
Yup, thank you.